app/containers/RoomItem/index.tsx

```diff
@@ -38,7 +38,7 @@ const RoomItemContainer = React.memo(
 		const testID = `rooms-list-view-item-${name}`;
 		const avatar = getRoomAvatar(item);
 		const isRead = getIsRead(item);
-		const date = item.roomUpdatedAt && formatDate(item.roomUpdatedAt);
+		const date = item?.lastMessage?.ts && formatDate(item.lastMessage.ts);
```

Comment:
This is a pseudo value and it's used as display here, but also as sort somewhere else.
If it's wrong this way, we should probably change the formula https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/1b7ab31c73cd4216d75f8d6b8cb18f681ae09d20/app/lib/methods/helpers/mergeSubscriptionsRooms.ts#L45

Reply:
Makes sense, will do. Thanks!